zebra/kernel_socket.c

```diff
@@ -1463,6 +1457,14 @@ void kernel_terminate(struct zebra_ns *zns, bool complete)
 	return;
 }
 
+/*
+ * Called by the dplane pthread to read incoming OS messages and dispatch them.
+ */
+int kernel_dplane_read(struct zebra_dplane_info *info)
+{
+	return 0;
```

Comment:
Why is this empty? Are we intentionally ignoring the OS messages?

Reply:
this first round of work is linux/netlink only - I tried to mention that in the description at the top.

Reply:
Ohh yes. Got it.